css/style.css

```diff
@@ -22,24 +22,26 @@ main {
     display: grid;
     grid-template-columns: 1fr;  /* 1 column layout on small screens */
     margin: 0.3em;
```

Comment:
Should this margin go away then?

Reply:
I kept it for now.

Reply:
No, that's the margin around the main element and resets the `margin: 0` in the body (done for the footer).